system/modules/core/classes/FrontendTemplate.php

```diff
@@ -251,7 +251,7 @@ protected function addToCache()
 			$strHeader = sprintf
 			(
 				"<?php /* %s */ \$expire = %d; \$content = %s; \$type = %s; \$files = %s; \$assets = %s; ?>\n",
-				$strCacheKey,
+				ltrim($strCacheKey, '*'),
```

Comment:
Sure, that should work too :). Should even work without the line breaks, I think.